src/components/SEO/SEO.jsx

```diff
@@ -1,9 +1,9 @@
-import React, { Component } from "react";
+import React from "react";
 import Helmet from "react-helmet";
 import urljoin from "url-join";
 import config from "../../../data/SiteConfig";
 
-class SEO extends Component {
+export default class SEO extends React.Component {
```

Comment:
Is there a reason why you use `React.Component` instead of just `Component`?

Reply:
We are already importing the whole `react` system, so instead of importing `Component` too from React separately we are just using dot operator to access it. So it's nothing special but more of a personal opinion.